src/main/java/alfio/manager/payment/BaseStripeManager.java

```diff
@@ -240,7 +240,7 @@ private BalanceTransaction retrieveBalanceTransaction(String balanceTransaction,
 
     Optional<String> getConnectedAccount(PaymentContext paymentContext) {
         if(isConnectEnabled(paymentContext)) {
-            return configurationManager.getStringConfigValue(paymentContext.narrow(STRIPE_CONNECTED_ID));
+            return configurationManager.getFor(paymentContext.getEvent(), STRIPE_CONNECTED_ID).getValue();
```

Comment:
same as before, potential NPE

Reply:
fixed in the new commit, if event is null, it will fetch the configuration at system level